websites/T/Tempest Fansub/presence.ts

```diff
@@ -1,38 +1,41 @@
 const presence = new Presence({
 		clientId: "1047102386478534727",
 	}),
-	browsingTimestamp = Math.floor(Date.now() / 1000);
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	defaultLogo =
+		"https://cdn.rcd.gg/PreMiD/websites/T/Tempest%20Fansub/assets/logo.png",
+	ecchilogo = "https://imgur.com/BtQLEdY.png";
```

Comment:
Please implement this with a const enum Assets.

```suggestion
	browsingTimestamp = Math.floor(Date.now() / 1000);
	
	const enum Assets {
	Logo = "https://cdn.rcd.gg/PreMiD/websites/T/Tempest%20Fansub/assets/logo.png",
	EcchiLogo = "https://imgur.com/BtQLEdY.png",
```